google/cloud/aiplatform/training_jobs.py

```diff
@@ -2535,6 +2536,15 @@ def __init__(
     ):
         """Constructs a AutoML Tabular Training Job.
 
+        Example usage:
+
+        job = training_jobs.AutoMLTabularTrainingJob(
+            display_name="my_display_name",
+            optimization_prediction_type="classification",
+            optimization_objective="minimize-log-loss",
+            column_specs=my_column_specs,
```

Comment:
Extending this argument like so, can give users a quick idea of how to format column_specs:
 ```suggestion
            column_specs={ "column_1": "auto", "column_2": "auto"},
```